### DISCUSSION TITLE
Book Giveaway Request

### DISCUSSION THREAD

**Discussion by tenythomas01** [microprofile/microprofile-marketing#639] [Ideas]
Hi Team, 

I’m reaching out to you on behalf of Packt. We are one of the leading UK tech publishers and have published more than 7000+ books, articles, and videos.  

Packt is growing its partnerships with communities and we’d like to work with Microprofile to help the community unlock new opportunities and skills and to be a driving force into the lives of people who are constantly evolving with the passion to learn tech. 

I had a chance to view one of your tweets(https://twitter.com/MicroProfileIO/status/1468264499312578560) on a recent book published by Packt. Many thanks for putting up the tweet.
 
I'm interested to know if the Microprofile team would like to do a giveaway promotion for the title- Practical Cloud-Native Java Development with MicroProfile from Microprofile social accounts. We are happy to support the giveaway promotion with 5 print and 10 eBook copies. Please let me know your thoughts.

We also have our $5 campaign running where all our eBooks(including Practical Cloud-Native Java Development with MicroProfile from Microprofile) and videos are offered to the community at just $5.

Looking forward to hearing from you soon.

Please feel free to reach out to me at tenyt@packt.com.

**Comment by aeiras** [comment]
Hola @tenythomas01, 

I found you in twitter: https://twitter.com/TenyMariya

It is wonderful to meet you virtually. Thank YOU & the Packt Team for choosing to reach to MP via the git ticket. Amazing open source form and truly at the MP 's 7 core pilars.  

MP's goal is always to welcome collaboration with those in our communities, who wish to add value to our Java ecosystem and beyond. 

My name is Amelia. Caitlin @caitmm43 and I are the December and January Stewards yet all MicroProfilers can provide feedback on things like this.   You are most welcomed to pull Caitlin & me via comments. 

--- 
**Yes**- we can use MP media to promote the 4 MicroProfiler's book. That would be pretty fantastic in addition to the monthly recurring media pushes that the book has underway that include 2022 coverage. 

**What is required for readers to win the latest MP book?**

- 5 free copies
- 10 ebooks

- $5 campaign on all Packt books sounds valuable for knowledge access at a healthy price as well. 

- What are the requirements for such media pushes from your side?
- what are the deadlines for the start and end of each of the giveaways?
- the design cards for media communications (we could create diverse cards with Packt logo(s) and your feedback. Full collaboration on messaging, design, etc  
- what else should we consider on this partnership?

--  **Holiday Vacation & 2021**

- Most of us go into vacation at the end of tomorrow- global contributors, diverse hours. 
- It is likely that we won't get to execute the media promotion in 2021.
- Most of us return to MP contributing the first week of January

 **2022 collaboration**

- re-starts 1st week
-  **Recommendation** on how to move forward: each give away needs in own ticket with requirements, etc to avoid ping-pong & decrease miscommunication. 
- Though we can do all the work via this ticket and new tickets, you could also choose to join the MP Marketing call happening on January 17th, 10-11am PDT. Public and recorded to tell us more about this & future opportunities, introduce yourself, meet MicroProfilers, etc. No requirement to attend yet we welcome you.  Calendar access is public and welcoming to ALL: https://calendar.google.com/calendar/u/0/embed?src=gbnbc373ga40n0tvbl88nkc3r4@group.calendar.google.com&ctz=America/Los_Angeles

Wishing you good Holidays,

**Comment by caitmm43** [comment]
Hi Teny @tenythomas01, 

Found you on twitter as well :)  Thanks for sharing this awesome opportunity to collab!  

I've share this ticket with our marketing community forum, highlighting it as a priority discussion item for our next meeting Jan 17th.  As Amelia said, we welcome you to join the conversation.  

Until then, happy holidays!

**Comment by tenythomas01** [comment]
Hi Amelia and Caitlin,

Nice to e-meet you both. Compliments of the season!

Apologies for the delay in getting back to you. This email thread landed in my junk folder.

Emily Jiang helped me in reaching to the Microprofile team. We're happy to collaborate with the Microprofile team and giveaway a few copies of our recent book(Practical Cloud-Native Java Development with MicroProfile) to the community.

I look forward to receiving any updates after the meeting on 17th January.

Please let me know your thoughts on our responsibilities and if you need further details we can discuss it over a call.

Looking forward to hearing from you!


Kind regards,
Teny Thomas

________________________________

**Comment by caitmm43** [comment]
Teny,

No worries.  Hope your holidays were wonderful, and happy 2022!  That all sounds great.  We'll follow up after the 17th .  

@Emily-Jiang, thanks for making that happen!  Please add any updates / progress to this thread so everyone is in the loop.  Are you planning to continue owning this as it moves forward?   

- Caitlin

**Comment by Emily-Jiang** [comment]
@caitmm43 let's discuss this on 17th Jan at the marketing call. It will be good for @tenythomas01 to join the call as well. 
@tenythomas01 below is the call details. The time is GMT timezone.
Mon, January 17, 6pm – 7pm
Wherehttps://eclipse.zoom.us/j/96426400983 (map)
DescriptionZoom access:  https://eclipse.zoom.us/j/96426400983

Agenda: Agenda+Minutes public with edit access. Monthly volunteer stewards' lead the discussion. 

Process: Discussions are public, recorded, and exported to MP Marketing Youtube Playlist.

**Comment by tenythomas01** [comment]
Hi Caitlin,

Happy new year. Hope you had a good weekend!

Sure, we can catch up after 17th.

Hi Emily,

Hope you're well.

Many thanks you for sharing the call details. I follow IST, so the call will be at 11:30pm for me. I'll try to join the call. If not possible, I'll view the discussion on MP Marketing YouTube Playlist.

Please keep me posted on any updates.


Kind regards,
Teny Thomas

________________________________

**Comment by Emily-Jiang** [comment]
We had a call today to discuss this ticket. 
@tenythomas01 can you answer the following questions:
Q1: can you clarify the requirement for the ebook and print copy such as the restriction on the shipment location?
Q2: Can you confirm the review on amazon is a necessary requirement? 
Q3: Can we drop this requirement? We would like this requirement on amazon review to be dropped.
Q4: If conversion of 3 free copies to ebooks is possible, how many ebooks can we get additionally?

**Comment by tenythomas01** [comment]
Hi Emily,

Hope you're well.

Please find my answers to your questions below:

A1. With print books, we are restricted to the US, Canada, UK, and EU delivery only. If the recipient is not based in those territories, then we can send an eBook. To deliver a print copy we would need the following details of the recipient- address and phone number. And to send an eBook you can share the recipient's email address.

A2. Review on Amazon is not a compulsory requirement.

A3. Absolutely, we can.

Please let me know in case of any other concerns.


Kind regards,
Teny Thomas

________________________________

**Reply by Emily-Jiang** [reply]
@tenythomas01 can you also answer Q4:
Q4: If conversion of 3 free copies to ebooks is possible, how many ebooks can we get additionally?
Thanks

**Comment by caitmm43** [comment]
Thanks all, for continuing to move things forward on this ticket!  @Emily-Jiang, If you are open to remaining the point person on this ticket as monthly marketing stewards rotate for February, I'd like to designate you as an assignee.  If that's good with you, please accept.  If not feel free to un-assign.

**Reply by Emily-Jiang** [reply]
sure, thanks

**Comment by tenythomas01** [comment]
Hi Emily,

Hope you're well.

Yes, it's fine. We can give additional 5 eBook copies.


Kind regards,
Teny Thomas

________________________________

**Comment by Emily-Jiang** [comment]
We have chosen @ederks85 and Emerson for the hard copy winners. I have shared the delivery addresses with @tenythomas01 . Thank you @tenythomas01 !

**Comment by tenythomas01** [comment]
Thanks for sharing, Emily.


Kind regards,
Teny Thomas

________________________________
